src/libraries/Microsoft.Extensions.Logging.Console/ref/Microsoft.Extensions.Logging.Console.csproj

```diff
@@ -21,6 +21,11 @@
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Options\ref\Microsoft.Extensions.Options.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
+    <Reference Include="netstandard" />
```

Comment:
why do you need the `netstandard` reference here?

Reply:
Because we are referencing .NETStandard configurations in the P2Ps above which requires the netstandard.dll shim.